fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/rewrite/TopicRewriteJob.java

```diff
@@ -51,4 +51,9 @@ public void execute(JobContext jobContext) {
     public boolean isOnce() {
         return true;
     }
+
+    @Override
+    public String getRuleType() {
```

Comment:
why add this interface? u do not use it in anyplace

Reply:
used for trace